ci/check-manifest.sh

```diff
@@ -38,6 +38,7 @@ YAMLS=(
 rm "${YAMLS[@]}"
 make manifest
 diff="$(git status --porcelain ${YAMLS[@]})"
+echo $diff
```

Comment:
This line does not relate to this commit?

Reply:
This was for debugging and I forget deleting it